src/registrar/models/domain_application.py

```diff
@@ -571,7 +571,12 @@ def _send_status_update_email(self, new_status, email_template, email_template_s
 
         if self.submitter is None or self.submitter.email is None:
             logger.warning(f"Cannot send {new_status} email, no submitter email address.")
-            return
+            return None
+
+        if do_fake_send_email:
```

Comment:
I'm not confident that I understand `do_fake_send_email` means so I had 2 questions!

1. Does the flag indicate that an email can be sent for testing purposes but doesn't actually send the email?
2. Is it only used in the context of locally testing email delivery?

Reply:
Probably because of confusing variable naming on my part! "send_email" might be less confusing
1. Yup. When we first start the app and run fixtures (generating fake data), it will "approve" a predetermined number of domains. Since we send an email on approval, this means that we are trying to send out 20-40 real emails everytime we docker-compose up. This variable is used exclusively for fixtures, and instead of actually sending them, it will just make a log entry instead
2. No - the intend for the change is that will be used only in our fixtures file, but not anywhere else